flink-runtime/src/main/java/org/apache/flink/runtime/filecache/FileCache.java

```diff
@@ -99,7 +99,8 @@ public FileCache(String[] tempDirectories) throws IOException {
 		this.shutdownHook = createShutdownHook(this, LOG);
 
 		this.entries = new HashMap<JobID, Map<String, Tuple4<Integer, File, Path, Future<Path>>>>();
-		this.executorService = Executors.newScheduledThreadPool(10, ExecutorThreadFactory.INSTANCE);
+		this.executorService = Executors.newScheduledThreadPool(10, 
```

Comment:
Is there any rational behind the magic number 10? Or just because it has been 10 before?